### DISCUSSION TITLE
Alpha Ratio: Add additional systematics for alternate MC

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#43] [Comments_HIG_23_001]
Alpha Ratio: Add additional systematics for alternate MC

**Comment by jialin-guo1** [comment]
[AlphaUncertainty_2018.txt](https://github.com/ram1123/2l2Q_limitSettingTool/files/12384651/AlphaUncertainty_2018.txt)

**uncertainty  = abs(n_alpha_ptbin - n_alpha_M50)/n_alpha_ptbin**
